### PR TITLE
Ft/loading UI

### PR DESCRIPTION
**Pull request by mindsgn**
Fixes #

**Changes proposed in this pull request:**

- created loading animation.

### CODE REVIEW

**Comment by 0xSero** [comment]
Can't connect to the UI and there's this weird blue bar. 

image

**Comment by 0xSero** [comment]
@mindsgn are you still working on this or should I close PR?

**Comment by mindsgn** [comment]
removes the loading screen after data loads.
Feb-13-2023 16-39-31

**Comment by 0xSero** [comment]
image

@mindsgn the timing is correct but it's not covering the screen

**Comment by mindsgn** [comment]
> image
> 
> @mindsgn the timing is correct but it's not covering the screen

thats a bit odd. I will have a look at it.

**Comment by mindsgn** [comment]
@Seroxdesign the latest deploy covers the screen

**Review by 0xSero — APPROVED** [review]

**Comment by 0xSero** [comment]
@mindsgn approved

File: src/containers/main/HeaderContainer/index.tsx [hunk 1]
```diff
@@ -27,6 +27,10 @@ export const HeaderContainer: FC<IProps> = ({ address, balance }) => {
 		setOpen(!open);
 	}, [open, setOpen]);
 
+	React.useEffect(() => {
```
Comment (0xSero) [inline src/containers/main/HeaderContainer/index.tsx]: remove this

File: src/context/index.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import Logo from './../assets/images/logo.svg';
+
+export const LoadingContext = React.createContext({
+	loading: true,
+	setLoading: (loading: boolean) => {},
+});
+
+export const useLoading = () => React.useContext(LoadingContext);
+
+export const LoadingProvider = ({ children }: { children: any }) => {
```
Comment (0xSero) [inline src/context/index.tsx:11]: ReactComponent is the correct type right>

**Review by 0xSero — APPROVED** [review]

**Review by mikeghen — APPROVED** [review]

**Pull request merged**